### PR TITLE
Issue template fix

### LINKED ISSUE #19: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should have an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
**Contributor Resources**
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by bupd**
**Description**

This PR fixes #19 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -35,3 +35,4 @@ Steps to reproduce the behavior:
 ---
 **Contributor Resources**
 - See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

```

Comment (thisiskaransgit) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @bupd, update the description files too

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bupd) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks for the suggestion will fix that.

Comment (bupd) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @thisiskaransgit updated with the suggestion

**Review by bupd — COMMENTED** [review]
Waiting for the PR to be merged.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bupd) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Is there something that I am missing..?

**Review by thisiskaransgit — APPROVED** [review]
@bupd, Looks good to merge, thank you for your contribution :tada:

**Comment by bupd** [comment]
@thisiskaransgit  can we merge this PR #21 this fixes #19 issue
Looks good to merge.

**Pull request merged**